src/EditorFeatures/CSharpTest/Diagnostics/GenerateMethod/GenerateDeconstructMethodTests.cs

```diff
@@ -166,7 +168,7 @@ void Method()
 {
     private void Deconstruct(out object x, out object y)
     {
-        throw new System.NotImplementedException();
+        throw new NotImplementedException();
```

Comment:
For me this looks fine, but it would be better to check with the original test author to see the old behavior is not by design.